### DISCUSSION TITLE
Modelleerprincipes - Definitie metadata

### DISCUSSION THREAD

**Discussion by pmaria** [Geonovum/disgeo-imsor#50] [General] [answered]
De definitie van [metadata](https://geonovum.github.io/disgeo-imsor/modelleerprincipes/#dfn-registratiemetadata) luidt nu
> Metadata definiëren we in navolging van [Richtlijn-Metagegevens] gegevens over informatieobjecten
>
> In de Richtlijn Metagegevens Overheidsinformatie wordt metadata omschreven als: "Gegevens over informatieobjecten. Zij maken het mogelijk informatieobjecten te vinden, uit te wisselen en te beheren".

Tijdens de [laatste expertgroep meeting](https://github.com/Geonovum/disgeo-imsor/blob/master/overleg/27-05-2021-aantekeningen.md) hadden we hier discussie over en werd deze definitie als te beperkt gezien:
- "het gaat om 'gegevens over gegevens', niet [alleen over] 'gegevens over informatieobjecten' - anders zeg je dat je geen metadata kunt opnemen over de directe eigenschappen."
- je kunt metadata op een informatiemodel of onderdelen daarvan opnemen, er zijn meerdere soorten metadata

**Comment by lvdbrink** [comment]
i.e. metadata op het niveau van datasets; op het niveau van informatiemodel; op het niveau van gegevens. 

Ipv te spreken over 'metadata' is het wellicht beter te spreken van 'dataset beschrijving', 'informatiemodel beschrijving' en 'gegevens over gegevens'. Of iets dergelijks.

**Reply by pmaria** [reply]
Eens, als we de term metadata gebruiken zouden we het moeten specificeren als X-metadata.

Tijdens onze eerder verkenning op metadata hadden we de volgende soorten metadata onderscheiden:
* Gegeven-metadata: (meta)gegevens over een enkel gegegeven
* Informatieobject-metadata: (meta)gegevens over een informatieobject
* Informatiemodel-metadata: (meta)gegevens over een informatiemodel, of elementen daarvan

Verder onderscheidden we voor de SOR een specifieke subset van bovenstaande categorieen:  registratie-metadata

* Registratie-metadata: (meta)gegevens die te maken hebben met de registratie., bijv:
  * autorisatie
  * kwaliteitseisen
  * tijdsaspecten.
  * etc.

**Comment by lvdbrink** [comment, marked as answer]
Uit expertgroepmeeting 17/6: 

De definitie van metadata moet breder, want je hebt metadata op allerlei niveaus. Het is dus beter om een definitie op te nemen van alle soorten metadata die we hanteren.

**Comment by lvdbrink** [comment]
Nieuwe definitie waarin verschillende soorten metadata worden genoemnd is vastgesteld in [expertgroep 8 juli](https://github.com/Geonovum/disgeo-imsor/blob/master/overleg/08-07-2021-aantekeningen.md).
